templates/dialog/investigation/chooseEvidenceSkill.hbs

```diff
@@ -0,0 +1,8 @@
+<form class="{{cssClass}}" autocomplete="off">
+    <h1>{{localize "WITCHER.Investigation.evidence.chooseSkill"}}</h1>
+    <div>
+        <select name="selectedSkill">
+            {{selectOptions skills nameAttr='name' valueAttr="name" labelAttr="label" localize=true}}
```

Comment:
shouldn't the value be a technical name or an id?

Reply:
I dont understand the question